### ISSUE TITLE
Is that possible to use gstreamer hls plugin?

### ISSUE THREAD

**Issue by solderzzc** [swarmnyc/HLSConvertor#1]

**Comment by touren** [comment]
Absolutely, the project is pending now because I decide to use another solution instead of Kurento.

**Comment by solderzzc** [comment]
Oh, may I please know which is the solution better than Kurento?

**Comment by acroca** [comment]
I'm interested too, I'm trying to get a WebRTC -> HLS but I can't find any good-and-easy way to do it.

**Comment by touren** [comment]
Kurento Server is really good at handling the live streaming, but HLS is actually a web server that provide a video files list and let the player to download, cache, then play the video files one by one. So a web server, ngix is my choice, should be much better than Kurento to do HLS.
This link should be helpful:
https://www.vultr.com/docs/setup-nginx-on-ubuntu-to-stream-live-hls-video

**Comment by AtheMathmo** [comment]
I hope you don't mind me jumping in...

If you are not using Kurento then how are you transcoding the WebRTC to RTMP for the nginx module?

**Comment by touren** [comment]
@AtheMathmo You are absolutely right. What I mean is not using this HLSConvertor(designed to be a Kurento Plugin) to do HLS. We only need an off-the-shelf Kurento Server to decode WebRTC.

**Comment by AtheMathmo** [comment]
@touren Ah ok, thanks for clarifying.

We've been considering a similar approach but are still trying to figure out the best way to do the RTP -> RTMP mapping required for the nginx module. If you have any insights you can share it would be appreciated!

**Comment by touren** [comment]
@AtheMathmo I used ffmpeg to do RTP->RTMP, because of comfortability.

**Comment by CheyenneForbes** [comment]
Hi @touren  could you provide the steps to use it with the gstreamer HSL plugins?
